validation-test/stdlib/XCTest.swift

```diff
@@ -252,6 +252,75 @@ XCTestTestSuite.test("XCTAssertThrowsError") {
 
 }
 
+XCTestTestSuite.test("XCTAssertNoThrow") {
+    class ErrorTestCase: XCTestCase {
```

Comment:
Another indentation fix.  We try to use 2-space indents here.  The other Swift-related projects may have different styles.